src/LibraryWindow.vala

```diff
@@ -452,7 +452,13 @@ public class Noise.LibraryWindow : LibraryWindowInterface, Gtk.Window {
         icon_name = "multimedia-audio-player";
         title = _("Music");
 
-        set_default_size (App.saved_state.get_int ("window-width"), App.saved_state.get_int ("window-height"));
+        var window_height = App.saved_state.get_int ("window-height");
+        var window_width = App.saved_state.get_int ("window-width");
+        var rect = Gtk.Allocation ();
+        rect.height = window_height;
+        rect.width = window_width;
+        set_allocation (rect);
```

Comment:
Documentation says that `set allocation ()` should only be used inside a widget's `size_allocate ()` method.  But I am not sure what is wrong with set_default_size () for a window anyway?